scipy/special/xsf/zeta.h

```diff
@@ -79,10 +99,13 @@ namespace detail {
 	if (n < 50) {
 	    result = zeta_em_log_abs_coeff_lookup[n];
 	} else {
-	    // asymptotic formula
-	    result = std::log(2.0) - 2.0*(n + 1)*std::log(2*M_PI);
+	    /* Asymptotic formula
+	     * Uses https://dlmf.nist.gov/24.11#E1 to approximate B_{2n} and
+	     * Stirling's approximation for (2n)!.
+	     */
+	    result = std::log(2.0) - 2.0*n*std::log(2*M_PI);
```

Comment:
Ah, great. That's much easier to see when looking at the right asymptotic approximation!